src/elli_prometheus_config.erl

```diff
@@ -0,0 +1,32 @@
+-module(elli_prometheus_config).
+
+-export([path/0,
+         format/0,
+         duration_buckets/0]).
+
+-define(DEFAULT_PATH, <<"/metrics">>).
+-define(DEFAULT_FORMAT, prometheus_text_format).
+-define(DEFAULT_DURATION_BUCKETS, [10, 100, 1000, 10000, 100000, 300000, 500000,
```

Comment:
Let me quote golang client docs here:

> // DefBuckets are the default Histogram buckets. The default buckets are
	// tailored to broadly measure the response time (in seconds) of a
	// network service. Most likely, however, you will be required to define
	// buckets customized to your use case.

https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go#L59

This buckets come from my personal experience basically.